apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/ApiCallRetryExtensions.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Api.Gax.Grpc;
+using Grpc.Core;
+using System;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    internal static class ApiCallRetryExtensions
+    {
+        // TODO: This is a modified/simplified version of ApiCallRetryExtensions.WithRetry from Google.Api.Gax.Grpc. Can we combine them somwhow?
```

Comment:
Nit: somwhow  => somehow :)

But yes, it would be good to remove this duplication. Suggest we look at that again after all the other refactoring is done?